.github/workflows/build_release.yaml

```diff
@@ -422,56 +152,18 @@ jobs:
           GH_TOKEN: ${{ github.token }}
         run: |
             BINARIES=($(find ./releases -type f))
-            gh release create --prerelease prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
-              --target ${{ github.event.inputs.ref }} --title prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
+            gh release create \
+              --prerelease prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
+              --target ${{ github.event.inputs.ref }} \
+              --title prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
               "${BINARIES[@]}"
 
-      # to-do rename step after depcreation - matter-labs/zksolc-bin
       - name: Publish release (this repository)
-        if: (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v'))
+        if: github.ref_type == 'tag'
         env:
           GH_TOKEN: ${{ github.token }}
         run: |
             BINARIES=($(find ./releases -type f))
             gh release create ${{ steps.version_or_sha.outputs.version_or_sha }} \
               --title ${{ steps.version_or_sha.outputs.version_or_sha }} \
               "${BINARIES[@]}"
-
-      - name: Publish release
```

Comment:
I'm removing this because the way it's implemented here will never be supported for open-source repositories, as this user was available only for private repositories. For now, we have to publish manually or write a new workflow in the *bin repositories later.